meta/saisanitycheck.c

```diff
@@ -1856,9 +1861,10 @@ void check_attr_acl_fields(
         case SAI_ATTR_VALUE_TYPE_ACL_ACTION_DATA_OBJECT_ID:
         case SAI_ATTR_VALUE_TYPE_ACL_ACTION_DATA_OBJECT_LIST:
 
-            if (md->objecttype != SAI_OBJECT_TYPE_ACL_ENTRY ||
-                    md->attrid < SAI_ACL_ENTRY_ATTR_ACTION_START ||
-                    md->attrid > SAI_ACL_ENTRY_ATTR_ACTION_END)
+            if (md->objecttype != SAI_OBJECT_TYPE_ACL_ENTRY  ||
+                    ((md->isextensionattr == false) &&
```

Comment:
can you bring back previous condition and add this one before it:

```
if (md->objecttype != SAI_OBJECT_TYPE_ACL_ENTRY && metadata->isextensionattr)
{
    /* allow acl action data on excetnsion acl etnry attributes */
    break;
}
```

i think this will make code more readable

Reply:
Done